keyboards/giabalanaipico/config.h

```diff
@@ -0,0 +1,104 @@
+/* Copyright 2022 3araht
+ *
+ * This program is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License as published by
+ * the Free Software Foundation, either version 2 of the License, or
+ * (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with this program.  If not, see <http://www.gnu.org/licenses/>.
+ */
+
+#pragma once
+
+/* Handedness by Pin */
+/* pull-up: left side,   pull-down: right side */
+#define SPLIT_HAND_PIN GP28
+
+/* test matrix scan speed */
+// #define DEBUG_MATRIX_SCAN_RATE
+
+// Right side has to be the master since 1, LED data is output from right side, and 2, Audio pin is prepared on right side as a reserve.
+#define MASTER_RIGHT
+
+// 2022/03/28 added. This might solve the "no left side communication via TRRS" issue.
+// #define SPLIT_USB_DETECT
+// SPLIT_USB_DETECT was not necessary since it is "Enabled by default on ChibiOS/ARM."
+/* (RP2040) Speedup boot sequence by avoiding to use SPLIT_USB_DETECT */
```

Comment:
If vbus pin is defined, split usb detect isn't used. 
```suggestion
```

Reply:
Done.